index.html

```diff
@@ -85,160 +88,22 @@ <h4 >Tonic</h4>
-                        <li>
-                            html
-                        </li>
-                        <li>
-                            css
-                        </li>
-                        <li>
-                            javascript
-                        </li>
-                    </ul>
-                    <div class="action-div">
-                        <button class="secondary-btn"><p>See Project</p></button>
-                    </div>
-                </div>
-
-            </div>
-            <div class="card">
-                <img src="./img/snapshoot-portfolio2.png" alt="Avatar">
-                <div class="work-description">
-                    <div class="work-title">
-                        <h4 >Tonic</h4>
-                        <ul class="work-info">
-                            <li class="client">
-                                <p>
-                                    Canopy
-                                </p>
-                            </li>
-                            <li class="role">
-                                <p>
-                                    Back End Dev
-                                </p>
-                            </li>
-                            <li class="year">
-                                <p>
-                                    2015
-                                </p>
-                            </li>
-                        </ul>
-                    </div>
-                    <div class="work-content-text">
-                        <p>
-                            A daily selection of privately personalized reads; no accounts or sign-ups required.
-                        </p>
-                    </div>
-                    <ul class="work-tech-used">
-                        <li>
-                            html
-                        </li>
-                        <li>
-                            css
-                        </li>
-                        <li>
-                            javascript
-                        </li>
-                    </ul>
-                    <div class="action-div">
-                        <button class="secondary-btn"><p>See Project</p></button>
+                    <img class="mod-img" src="" alt="Avatar">
+                    <div class="mod-desc-content">
+                        <p class="mod-desc">
+                        </p> 
+                        <div class="mod-lang-btn">
+                            <ul class="mod-lang work-tech-used">
+                            </ul>
+                            <div class="btn-contain">
+                                <button class="btn-mod"><a class="mod-live" href="">See Live <img src="./img/live.png"></a></button>
+                                <button class="btn-mod"><a class="mod-source" href="">See Source <img src="./img/git.png"></a></button>
+                            </div>
+                        </div>
```

Comment:
- [ ] As per the project guidelines, since this section should be rendered dynamically, I would suggest that you ship these codes for the Modal in a javascript file where you render the project dynamically also. This means that here in Html, there will not be anything concerning the projects section since the whole section will be rendered dynamically 😃 👍